src/TMap.cpp

```diff
@@ -1335,83 +1335,21 @@ bool TMap::restore( QString location )
-                                                      "from this by going to the \"map\" sub-directory of this profile's home\n"
-                                                      "directory and selecting a file to load that is NOT the most recent one\n"
-                                                      "(which is the one that is selected normally) though it is probable that\n"
-                                                      "the stored location of the current map location will be wrong." );
-                    appendErrorMsgWithNoLf( infoMsg );
-                    postMessage( infoMsg );
-                    file.close();
-                    return false;
-                }
-                else {
-                    QString alertMsg = tr( "[ ALERT ] - Map file is using new features, it's file format (%1) is higher than\n"
-                                                       "the default for this version of Mudlet (%2)!" )
-                                     .arg( mVersion )
-                                     .arg( mDefaultVersion );
-                    appendErrorMsgWithNoLf( alertMsg );
-                    postMessage( alertMsg );
-                    QString infoMsg = tr( "[ INFO ]  - You are using a new, possibly experimental, map file format which this\n"
-                                                      "version of Mudlet CAN read.  This Mudlet appears to be based on a\n"
-                                                      "development version of the source code so it is possible that you, or the\n"
-                                                      "creator of the Map file, may have been experimenting with this more\n"
-                                                      "advanced version that has additional features that need a revised format;\n"
-                                                      "beware that this file may not be usable by the current release version\n"
-                                                      "of Mudlet that you or others with whom you might share it have!" );
-                    appendErrorMsgWithNoLf( infoMsg );
-                    postMessage( infoMsg );
-                    mSaveVersion = mVersion; // Make the save version default to the loaded one
-                                             // this means that each session using a particular
-                                             // map file version will continue to use it unless
-                                             // the user intervenes.
-                }
-            }
-        }
-        else if( mVersion < 4 ) {
+        if( mVersion > mMaxVersion ) {
+            QString errMsg = tr( "[ ERROR ] - Map file is too new, it's file format (%1) is higher than this version of\n"
+                                 "Mudlet can handle (%2)!  The file is: \"%3\"." )
```

Comment:
Can I request a line feed before the file name is shown - in hindsight the absolute path name can get a bit long and - with the 10 to 12 character indent it would reduce the chance of wrapping the output line, i.e. change this line to:

    "Mudlet can handle (%2)!  The file is:\n\"%3\"." )